components/collective-page/hero/Hero.js

```diff
@@ -67,10 +68,27 @@ const StyledShortDescription = styled.h2`
 const Hero = ({ collective, host, isAdmin, onPrimaryColorChange, callsToAction, intl }) => {
   const [hasColorPicker, showColorPicker] = React.useState(false);
   const [isEditingCover, editCover] = React.useState(false);
+  const [message, showMessage] = React.useState(null);
   const isEditing = hasColorPicker || isEditingCover;
   const isCollective = collective.type === CollectiveType.COLLECTIVE;
 
-  return (
+  const handleHeroMessage = msg => {
+    if (!msg) {
+      showMessage(null);
+    } else {
+      showMessage({
+        type: msg.type || 'info',
+        content: msg.content || msg,
+      });
+    }
+  };
+
+  return [
```

Comment:
I used this syntax instead of Fragment to minimize the diff. Not sure if it's nice or wrong :-)

Reply:
Array items must normally be keyed (https://reactjs.org/docs/jsx-in-depth.html#jsx-children), I'm not sure if that can lead to errors or if React is now clever enough to prevent this